examples/stable-diffusion/train_text_to_image_sdxl.py

```diff
@@ -1204,8 +1208,12 @@ def compute_time_ids(original_size, crops_coords_top_left):
                         accelerator.save_state(save_path)
                         logger.info(f"Saved state to {save_path}")
 
-            logs = {"step_loss": loss.detach().item(), "lr": lr_scheduler.get_last_lr()[0]}
-            progress_bar.set_postfix(**logs)
+            if global_step % args.logging_step == 0:
+                train_loss_scalar = train_loss.item()
+                accelerator.log({"train_loss": train_loss_scalar}, step=global_step)
+
+                logs = {"step_loss": train_loss_scalar, "lr": lr_scheduler.get_last_lr()[0]}
```

Comment:
we can change here
if args.accumulation_steps > 1:
     logs = {"step_loss": loss.item(), "lr": lr_scheduler.get_last_lr()[0]}
else:
    logs = {"step_loss": train_loss_scalar, "lr": lr_scheduler.get_last_lr()[0]}